libs/ui/util/zindex.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright Oxide Computer Company
+ */
+
+export const zIndex = {
```

Comment:
![IntelliSense with the assist](https://github.com/oxidecomputer/console/assets/22547/edb6ee59-8008-4c5d-8a44-7394bbd8d818)